### PR TITLE
Triggering wallet connection from trade page on submission

### PR DESCRIPTION
**Pull request by alfetopito**
Follow up to #588 

When user is logged and tries to submit an order, the wallet dropdown is shown over the trade page.

### Notes
- ~~There's a bug on `onSubmit` where the form data is not being passed down onto `data` object. To test I manually set the values. I guess this will be fixed by https://github.com/gnosis/dex-react/pull/551 so I didn't spend time trying to fix it. For this reason, you'll have to trust me it works or test locally.~~
- ~~It seems like the wallet widget is re-rendered every second, guess due to the batch time remaining thingy. Still have to check that.~~ Yep, that's the case.
- ~~For a reason that I yet haven't figured out, the wallet dropdown is expanded after logging in.~~

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr598--dexreact.review.gnosisdev.com

File: src/components/TradeWidget/index.tsx [hunk 1]
```diff
@@ -461,8 +461,8 @@ const TradeWidget: React.FC = () => {
         dispatch(removePendingOrdersAction({ networkId, pendingTxHash, userAddress }))
       }
     } else {
-      const from = history.location.pathname + history.location.search
-      history.push('/connect-wallet', { from })
+      // Not connected. Prompt user to connect his wallet
+      await connectWallet()
```
Comment (Velenir) [inline src/components/TradeWidget/index.tsx]: So it only asks to connect, but doesn't submit the transaction after connection
Kinda not intuitive

Comment (W3stside) [reply src/components/TradeWidget/index.tsx]: this is where redux middleware would be very useful

Comment (Velenir) [reply src/components/TradeWidget/index.tsx]: can just send the tx if connection is successful

Comment (alfetopito) [reply src/components/TradeWidget/index.tsx]: Would you rather do that after connecting?

Comment (Velenir) [reply src/components/TradeWidget/index.tsx]:
```js
if (!isConnected) //send tx
else {
  // connect
  if (success) // send tx
}
```

would be my preference
@anxolin , @W3stside, @biocom  ?

Comment (W3stside) [reply src/components/TradeWidget/index.tsx]: yes i think it makes the most sense :100:

**Pull request merged**